migrations/config-test.json

```diff
@@ -1,10 +1,13 @@
 {
-  "ownerAddress":"0x90f8bf6a479f320ead074411a4b0e7944ea8c9c1",
-  "sysAddress":"0x1b0b9453b425b4fe289bef5d00eaf9a4e173b957",
-  "preIcoAddress":"0x9f1b08fe54278cc55b381f4a95516b87c10cb38e",
-  "walletAddress": "0x2A3362bDF864915C2eb996cEE868a841586F4aD4",
-  "bountyWalletAddress":"0xca5DA912c9638856C69eb3a00eBea073142B0315",
-  "emtWithdrawalAddress": "0xc7FE2c59904efB1534C83f5A04C4e8306d0b7086",
+  "ownerAddress":"0xee676e153eb40133c7dc310501ef14c6469c30a6",
+  "sysAddress":"0xb7ee1b8491d248926aafaa8297b52f6c79da2a84",
+  "reservationAddress":"",
```

Comment:
I prefer to use (ignored by git) config.json which everyone should create individually.